lib/rules/no-nested-ternary.js

```diff
@@ -11,6 +11,8 @@
 
 module.exports = {
     meta: {
+        type: "style",
```

Comment:
Since fixing errors reported by this rule would require refactoring the code (albeit said refactoring is tiny), should this be a "suggestion"?

Reply:
Yeah, I'd tend to agree.